guides/common/modules/proc_associating-the-dhcp-service-with-a-subnet.adoc

```diff
@@ -0,0 +1,14 @@
+[id="associating-the-dhcp-service-with-a-subnet"]
+= Associating the DHCP service with a subnet
+
+After you configured or changed the DHCP provider, you must update the configuration of each affected {SmartProxy} in the {ProjectWebUI}.
+
+.Prerequisites
+* You configured a DHCP provider.
```

Comment:
```suggestion
```
Because this prereq is already included in line 4, isn't it?

Reply:
Since no prereq was left, I removed the entire Prereqs sub-section.